spec/metric_fu/metrics/reek/generator_spec.rb

```diff
@@ -48,46 +51,53 @@
-                   lines: [6, 9])
+            instance_double(Reek::Smells::SmellWarning,
+                            source: "app/controllers/activity_reports_controller.rb",
+                            context: "ActivityReportsController#authorize_user",
+                            message: "calls current_user.primary_site_ids multiple times",
+                            smell_type: "Duplication",
+                            lines: [2, 4]),
+            instance_double(Reek::Smells::SmellWarning,
+                            source: "app/controllers/activity_reports_controller.rb",
+                            context: "ActivityReportsController#authorize_user",
+                            message: "calls params[id] multiple times",
+                            smell_type: "Duplication",
+                            lines: [5, 7]),
+            instance_double(Reek::Smells::SmellWarning,
+                            source: "app/controllers/activity_reports_controller.rb",
+                            context: "ActivityReportsController#authorize_user",
+                            message: "calls params[primary_site_id] multiple times",
+                            smell_type: "Duplication",
+                            lines: [11, 15]),
+            instance_double(Reek::Smells::SmellWarning,
+                            source: "app/controllers/activity_reports_controller.rb",
+                            context: "ActivityReportsController#authorize_user",
+                            message: "has approx 6 statements",
+                            smell_type: "Long Method",
+                            lines: [8]),
+
+            instance_double(Reek::Smells::SmellWarning,
+                            source: "app/controllers/application.rb",
+                            context: "ApplicationController#start_background_task/block/block",
+                            message: "is nested",
+                            smell_type: "Nested Iterators",
+                            lines: [23]),
+
+            instance_double(Reek::Smells::SmellWarning,
+                            source: "app/controllers/link_targets_controller.rb",
+                            context: "LinkTargetsController#authorize_user",
+                            message: "calls current_user.role multiple times",
+                            smell_type: "Duplication",
+                            lines: [8]),
+
+            instance_double(Reek::Smells::SmellWarning,
+                            source: "app/controllers/newline_controller.rb",
+                            context: "NewlineController#some_method",
+                            message: "calls current_user.<< \"new line\n\" multiple times",
+                            smell_type: "Duplication",
+                            lines: [6, 9])
         ]
-        @lines = double(smells: @smells)
+        @lines = instance_double(Reek::Core::Examiner, smells: @smells)
```

Comment:
so, this test will fail on Reek versions before the Reek::Examiner was renamed to Reek::Core::Examiner, but I'm ok with it for now.